api/build.js

```diff
@@ -59,21 +59,22 @@ export async function build({ state, config, cwd = process.cwd() }) {
     const LAZY_INTERMEDIATE = join(ESBUILD_OUTDIR, 'lazy.js');
     const LAZY_FINAL = join(CLIENT_OUTDIR, 'lazy.js');
 
+    const hydrateSupport =
+      MODE === 'hydrate'
+        ? 'import "@lit-labs/ssr-client/lit-element-hydrate-support.js";'
+        : '';
+
     // Create entrypoints for each file type
     if (existsSync(CONTENT_SRC_FILEPATH)) {
       writeFileSync(
         CONTENT_ENTRY,
-        `import "${require.resolve(
-          '@lit-labs/ssr-client/lit-element-hydrate-support.js',
-        )}";import Component from "${CONTENT_SRC_FILEPATH}";customElements.define("${NAME}-content",Component);`,
+        `${hydrateSupport}import Component from "${CONTENT_SRC_FILEPATH}";customElements.define("${NAME}-content",Component);`,
```

Comment:
we can't use require.resolve here as node resolves this as a node dependency and lit provides a different dep for node vs the browser. We want the browser one.